server/routes/statistics.js

```diff
@@ -120,6 +121,20 @@ var businessProcessQueryHandler = new QueryHandler([{
 	}
 }]);
 
+var getPeriods = function () {
+	var periods = ['inPeriod', 'today', 'thisWeek', 'thisMonth']
+	  , today = toDateInTz(new Date(), db.timeZone)
+	  , currentYear = new db.Date(today.getUTCFullYear(), 0, 1)
+	  , lastYearInRange = new db.Date(today.getUTCFullYear() - 5, 0, 1);
```

Comment:
I asked, and we established that last 6 years (including current) is totally fine solution. Otherwise with each year the table becomes less and less readable.